datafusion/expr-common/src/signature.rs

```diff
@@ -343,6 +343,12 @@ impl TypeSignature {
     pub fn supports_zero_argument(&self) -> bool {
```

Comment:
I actually think we could remove this function entirely as it is only used to generate errors